nesta/production/batchables/unsdg/unsdg_text_predict/run.py

```diff
@@ -30,10 +35,21 @@ def dummy_model(text):
 
     return predictions
 
+def predict_sdg_labels(topic_dist, model_dictionary):
```

Comment:
In general, it might make more sense to use the keys of the model dictionary as the returned sdg labels. This would protect against the dictionary being accidentally created with the keys in the wrong order.